### PR TITLE
Create Caluculator Component

### PR DESCRIPTION
**Pull request by aeh1707**
- Create Calculator component along with sub-components namely Button and Screen components.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,26 +1,12 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
   return (
     <div className="App">
       <header className="App-header">
```
Comment (DammyShittu) [inline src/App.js]: - [x] All components should be class-based. Ensure that your App component is class-based just like the Calculator component.

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @aeh1707,

✔️ Calculator design matches wireframe

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- [x] Kindly delete the marked boilerplate files as they would not be necessary for this project
 
  delete-cra

- [ ] _Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (aeh1707) [reply src/App.js]: @DammyShittu Thanks I appreciate your precision

**Review by uwadonat — APPROVED** [review]
## Hi @aeh1707,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
- Well organized and clean project ✔️
- Short and clear commit messages ✔️
cong

# Status: Approved ✔️✔️✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- To add a title on your next pull request message. It looks more professional.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @uwadonat in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**